lib/core_dom/resource_url_resolver.dart

```diff
@@ -160,13 +160,11 @@ class ResourceUrlResolver {
     // If it's not absolute, then resolve it first
     Uri resolved = baseUri.resolve(uri);
 
-    // If it's package-relative, tack on 'packages/' - Note that eventually
-    // we may want to change this to be '/packages/' to make it truly absolute
+    // If it's package-relative, tack on '/packages/'
     if (resolved.scheme == 'package') {
-      return 'packages/${resolved.path}';
+      return '/packages/${resolved.path}';
```

Comment:
Tracked this down. The TEST_SERVER_BASE_PREFIX is used for any tests that are using type relative URIs. If you look at all the tests it is used they all have `@Component`s which define relative `templateUrl`s. This change makes all type relative URIs into absolute URIs which are default turned on for angular.